app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -94,6 +94,28 @@ const MapImpl: React.FunctionComponent = () => {
             transmissionPowerString: feature.transmissionPower
               ? feature.transmissionPower.toString() + " dB"
               : undefined,
+            Y:
```

Comment:
Or just use my parsedWkt from the server? Can do it another PR.

Reply:
If you're going to do this in the client at least use a regular expression, this code is hideous. You only need to call the regex once, too.